netctl/netctl.go

```diff
@@ -65,8 +65,6 @@ func inspectPolicy(ctx *cli.Context) {
 	tenant := ctx.String("tenant")
 	policy := ctx.Args()[0]
 
-	fmt.Printf("Inspeting policy: %s tenant: %s\n", policy, tenant)
```

Comment:
I would consider changing these to `log.Debugf` so they're available if you `DEBUG=1` but won't show up in normal usage